techpack/audio/asoc/msm8952.h

```diff
@@ -0,0 +1,92 @@
+/* Copyright (c) 2018-2019, The Linux Foundation. All rights reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 and
+ * only version 2 as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+#ifndef __MSM8953_H
```

Comment:
ping @kholk

Reply:
thanks, but don't worry, that's fine :D

Reply:
oops, yes:)
https://source.codeaurora.org/quic/la/platform/vendor/opensource/audio-kernel/tree/asoc/msm8952.h?h=LA.UM.7.8.r3-01400-SDM710.0#n13